Assets/HoloToolkit/Utilities/Prefabs/SceneLauncher.prefab

```diff
@@ -0,0 +1,319 @@
+  m_GameObject: {fileID: 1289556740807306}
+  m_Text: HoloToolkit Scenes
+  m_OffsetZ: 0
+  m_CharacterSize: 1
+  m_LineSpacing: 1
+  m_Anchor: 4
+  m_Alignment: 0
+  m_TabSize: 4
+  m_FontSize: 100
+  m_FontStyle: 0
+  m_RichText: 1
+  m_Font: {fileID: 12800000, guid: 86574c70442309b45be5a1c37a37a40b, type: 3}
+  m_Color:
+    serializedVersion: 2
+    rgba: 4293322470
+--- !u!102 &102639908367590550
+TextMesh:
+  serializedVersion: 3
+  m_ObjectHideFlags: 1
+  m_PrefabParentObject: {fileID: 0}
+  m_PrefabInternal: {fileID: 100100000}
+  m_GameObject: {fileID: 1177239735944006}
+  m_Text: 'Air tap a scene to load it.
+
+    When finished with a scene, say "Scene Launcher" or press the Backspace key to
+    return the this scene list.
+
+    Edit the scenes in this list in Unity > File > Build Settings > Scenes in Build.
+
+    Make sure that this scene (SceneLauncher) is the first scene (#0).
+
+    If you would like to use all scenes in the project, search for t:Scene in the
+    Project tab, select all, and drag all scenes into Scenes in Build.'
+  m_OffsetZ: 0
+  m_CharacterSize: 1
+  m_LineSpacing: 1
+  m_Anchor: 4
+  m_Alignment: 0
+  m_TabSize: 4
+  m_FontSize: 32
+  m_FontStyle: 0
+  m_RichText: 1
+  m_Font: {fileID: 12800000, guid: 86574c70442309b45be5a1c37a37a40b, type: 3}
+  m_Color:
+    serializedVersion: 2
+    rgba: 4293322470
+--- !u!114 &114087985097670268
+MonoBehaviour:
+  m_ObjectHideFlags: 1
+  m_PrefabParentObject: {fileID: 0}
+  m_PrefabInternal: {fileID: 100100000}
+  m_GameObject: {fileID: 1247193059148626}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: c62fd7a55ba7458499b69cae18f83806, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
+  PersistentKeywords: 1
+  RecognizerStart: 0
+  RecognitionConfidenceLevel: 1
+  Keywords:
+  - Keyword: Scene Launcher
+    KeyCode: 8
+--- !u!114 &114669922776097872
+MonoBehaviour:
+  m_ObjectHideFlags: 1
+  m_PrefabParentObject: {fileID: 0}
+  m_PrefabInternal: {fileID: 100100000}
+  m_GameObject: {fileID: 1247193059148626}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: f8d28210caf2b8340860c439c592e979, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
+  sceneMapping:
+  - ScenePath: Assets/HoloToolkit/Utilities/Scenes/HeadsetAdjustment.unity
+    IsButtonEnabled: 1
+  - ScenePath: Assets/HoloToolkit/Utilities/Scenes/SceneLauncher.unity
+    IsButtonEnabled: 1
+  ButtonSpawnLocation: {fileID: 1899756416056624}
+  SceneButtonPrefab: {fileID: 114712049552519426, guid: c943b5508bd87f347a6d36c5ca46424c,
+    type: 2}
+  MaxRows: 5
+--- !u!114 &114707455351036974
+MonoBehaviour:
+  m_ObjectHideFlags: 1
+  m_PrefabParentObject: {fileID: 0}
+  m_PrefabInternal: {fileID: 100100000}
+  m_GameObject: {fileID: 1247193059148626}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: e7d6513c2cdf97f409654a2a4114d9b1, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
+  Keywords:
+  - Keyword: Scene Loader
```

Comment:
This needs to be changed to Scene Launcher. Scene Loader is the old input source keyword that no longer fires.

Reply:
Huh, I thought I updated that, sorry!